2996_reflection/reflection.md

```diff
@@ -4073,6 +4433,40 @@ consteval size_t bit_size_of(info entity);
 :::
 :::
 
+### [meta.reflection.result] Expression result reflection {-}
+
+::: std
+::: addu
+```cpp
+template <typename T>
+  consteval info reflect_value(T expr);
+```
+
+[#]{.pnum} *Mandates*: `T` is a structural type. `T` is not a reference type. Any subobject of the value computed by `expr` having reference or pointer type designates an entity that is a permitted result of a constant expression.
+
+[#]{.pnum} *Returns*: A reflection of the value computed by an lvalue-to-rvalue conversion applied to `expr`. The type of the reflected value shall be `T` if `T` is of class type, and otherwise the cv-unqualified version of `T`. The type of the reflected value shall not be an alias.
+
+```cpp
+template <typename T>
+  consteval info reflect_object(T& expr);
+```
+
+[#]{.pnum} *Mandates*: `T` is not a function type. `expr` designates an entity that is a permitted result of a constant expression.
+
+[#]{.pnum} *Returns*: A reflection of the object whose identity is determined by the lvalue `expr`. If the reflected object is a variable `Obj`, the returned reflection shall compare equal to `^Obj`.
```

Comment:
As above with "shall":
```suggestion
[#]{.pnum} *Returns*: `^o`, where `o` is the object that `expr` references. 
```
But I think this is kind of a problem since you cannot write `^o` for some of those objects?

Reply:
I removed "shall", but you're right - I deliberately skirt that by referring to "the object whose identity is determined by the lvalue `expr`". Aside from the "shall", I the existing language is more exact.

Reply:
That said, I simplified the wording to be closer to what you suggested for `reflect_function`.